content/docs/user-guide/components/reference/atom/directional-light.md

```diff
@@ -13,8 +13,46 @@ The **Directional Light** component casts light from an infinitely distant point
 [Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
 
 
-## Properties
+## Directional Light Properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
+![Directional Light component interface.](/images/user-guide/components/reference/atom/light-component-ui/directional-light.png)
+
+| Property | Description | Values | Default |
+| - | - | - | - |
+| **Color** | The color of the light. The color acts as a mask or gel on perfectly white light source. For instance, a 1000 lux light with a white color will actually be 1000 lux, but a 1000 lux light with an 18% gray color will only be 180 lux. | Color | White |
+| **Intensity mode** | Allows specifying light values in lux or Ev100. Lux is a linear light value while EV100 is logarithmic (similar to camera stops). | `Lux` or `Ev100` | `Ev100` |

```

Comment:
```suggestion
| **Intensity mode** | Allows specifying light values in lux or EV100. Lux is a linear light value while EV100 is logarithmic (similar to camera stops). | `Lux` or `Ev100` | `Ev100` |
```
I have a couple edits here to make capitalization consistent when the lux and EV100 refers to the UI property vs. the unit.